### DISCUSSION TITLE
Repo question.

### DISCUSSION THREAD

**Discussion by macerg** [termux/termux-packages#15731] [Q&A]
Hello, not raising as an issue yet as may be my error.

When attempting an upgrade I am getting signature errors. I get the error regardless of which mirror I choose. This has been happening since yesterday's successful upgrade. See example:-

x11 InRelease [14.0 kB]
Err:1 https://packages.termux.dev/apt/termux-main stable InRelease
  At least one invalid signature was encountered.
Err:2 https://packages.termux.dev/apt/termux-root root InRelease
  At least one invalid signature was encountered.
Err:3 https://packages.termux.dev/apt/termux-x11 x11 InRelease
  At least one invalid signature was encountered.
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
All packages are up to date.
W: An error occurred during the signature verification. The repository is not updated and the previous index files will be used. GPG error: https://packages.termux.dev/apt/termux-main stable InRelease: At least one invalid signature was encountered.
W: An error occurred during the signature verification. The repository is not updated and the previous index files will be used. GPG error: https://packages.termux.dev/apt/termux-root root InRelease: At least one invalid signature was encountered.
W: An error occurred during the signature verification. The repository is not updated and the previous index files will be used. GPG error: https://packages.termux.dev/apt/termux-x11 x11 InRelease: At least one invalid signature was encountered.
W: Failed to fetch https://packages.termux.dev/apt/termux-main/dists/stable/InRelease  At least one invalid signature was encountered.
W: Failed to fetch https://packages.termux.dev/apt/termux-root/dists/root/InRelease  At least one invalid signature was encountered.
W: Failed to fetch https://packages.termux.dev/apt/termux-x11/dists/x11/InRelease  At least one invalid signature was encountered.
W: Some index files failed to download. They have been ignored, or old ones used instead.
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Calculating upgrade... Done
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.

**Comment by macerg** [comment]
Help in resolving this would be much appreciated.
Thanks.

**Comment by xtkoba** [comment]
See #15706.
